src/index.js

```diff
@@ -186,6 +186,12 @@ class Mplex extends EE {
     chan.once('close', () => {
       list[id] = null
     })
+    chan.once('error', (_) => {
+      // TODO: Should we handle this? Internally the channel
+      // is handling errors. In node 10+ if we dont catch this
+      // error, it will be thrown. Perhaps channel shouldnt be
+      // emitting errors?
+    })
```

Comment:
This was needed to fix an error with the test suite, because there was no error listener on channel. I need to do some more testing, but I don't think the channel needs to be emitting errors. It appears to be handling everything internally and reseting the stream properly, so I think it's probably redundant anyway.

Reply:
I think the Channel should be emitting errors, but those needs to be catched here and logged. A lot of channel erroring might be a smell than it's better to tear down the connection?